build.gradle

```diff
@@ -1,3 +1,5 @@
+import proguard.gradle.ProGuardTask
```

Comment:
not a blocker but i think this goes below the copyright header ideally?

Reply:
I'd agree, however that's where intellij put it :/ can move it i'd just worry intellijj would be weird next time it imports something. Alternatively we could use a canonical reference